Ginger/GingerCoreNET/GeneralLib/FormatTextTable.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET.GeneralLib
+{
+    public class FormatTextTable
+    {
+        List<list2strings> twoColumnList = new List<list2strings>();
```

Comment:
Need to have generic unlimited not predefined columns - need to rewrite

Reply:
Updated the class to be more generic